docs/tutorials/HD143006_Part_2.py

```diff
@@ -79,7 +79,7 @@
 
 # -
 
-# We now have everything from the last tutorial loaded and can begin the process of optimizing our model to set a better starting point.
+# We now have everything from the last tutorial loaded and can begin the process of optimization to improve our image quality and perform cross validation on our model to better determine hyperparameters.
```

Comment:
I think here we should probably just say we're starting the process of initializing the model and getting it closer to the dirty image since we don't get to cross validation until a while later in the tutorial. Also don't want to just restate what's happening the whole tutorial since  we already do that at the beginning.

Reply:
Good point, I agree

Reply:
I moved it to line 93 and omitted optimization and cross validation as they are in future sections